ProcessMaker/modeler
# Fix message flow undo/redo

Fixes #244.

**How to test:**
* Visit the issue linked above; ensure you can't reproduce the bug you that you experiecne the expected behaviour.
